public/locales/ja/common.json

```diff
@@ -20,32 +20,32 @@
   "likes": "いいね",
   "blog": "ブログ",
   "Owner": "所有者",
-  "Transaction Hash": "トランザクション ハッシュ",
-  "IPFS Address": "IPFS アドレス",
+  "Transaction Hash": "取引ハッシュ",
+  "IPFS Address": "IPF アドレス",
```

Comment:
Is this a typo?

Reply:
Absolutely, thank you for pointing out this problem. I have filed a new pr #208  and I will review my pr more carefully next time. sorry.